pgjdbc/src/test/java/org/postgresql/jdbc/ScramTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2021, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.jdbc;
```

Comment:
Would you please move it to `org.postgresql.test.jre8...` package? (then it would be easier to merge to 42.2 branch)